src/Features/LanguageServer/Protocol/Handler/Diagnostics/Public/PublicDocumentPullDiagnosticsHandler_IOnInitialized.cs

```diff
@@ -28,26 +29,24 @@ public async Task OnInitializedAsync(ClientCapabilities clientCapabilities, Requ
             // TODO: Hookup an option changed handler for changes to BackgroundAnalysisScopeOption
             //       to dynamically register/unregister the non-local document diagnostic source.
 
+            var sources = _diagnosticSourceManager.GetSourceNames(isDocument: true);
```

Comment:
@dibarbet is this correct? The comment mentions non-local source which I am not sure whether it is still needed